src/store/AuthStore.tsx

```diff
@@ -1,56 +1,31 @@
 import { create } from "zustand";
 import { persist } from "zustand/middleware";
 import { postSignIn } from "@/api/auth";
-import { postProfile } from "@/api/profile";
 import { AxiosError } from "axios"; // AxiosError import
 
-interface Profile {
-  updatedAt: string;
-  securityQuestion: string;
-  teamId: string;
-  content: string;
-  nationality: string;
-  family: string;
-  bloodType: string;
-  nickname: string;
-  birthday: string;
-  sns: string;
-  job: string;
-  mbti: string;
-  city: string;
-  image: string;
-  code: string;
-  name: string;
-  id: number;
-}
-
 interface User {
   id: number;
   email: string;
   name: string;
   teamId: string;
   updatedAt: string;
   createdAt: string;
-  profile: Profile | null;
+  profile: null;
```

Comment:
이러면 프로필에 null 값만 할당될 것 같아요! 수정해야 할 것 같습니다
```suggestion
  interface LoginProfile {
    id: number;
    code: string;
  }
  
  interface User extends LoginProfile {
    id: number;
    ...
  }
  // 혹은 profile: LoginProfile | null;
```

Reply:
앗 수정했습니다.